web/src/game/game-details.html

```diff
@@ -79,9 +82,10 @@
   </ghvz-super-form>
 
   <div hidden$="[[!game]]">
-    <div id="number">ID: <b>[[game.id]]</b></div>
-    <div id="name">Name: <b>[[game.name]]</b></div>
-    <div id="stunTimer">Stun Timer: <b>[[game.stunTimer]]</b></div>
+    <!-- Would it break anything to give the two elements diff names?-->
```

Comment:
not sure what you mean here, do you mean two elements the same name? or one element two names?

two elements with two different names sounds quite fine to me, lol